runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1420,8 +1421,15 @@ endfunc
 
 func s:GotoAsmwinOrCreateIt()
   if !win_gotoid(s:asmwin)
+    let mdf = ''
     if win_gotoid(s:sourcewin)
-      exe 'rightbelow new'
+      " 60 is approx spaceBuffer * 3
+      if winwidth(0) > (78 + 60)
+        let mdf = 'vert'
+        exe mdf .. ' ' .. 60 .. 'new'
```

Comment:
Instead of hard-coding the window width as 60, can you add a new configuration item to g:termdebug_config and use that here?  You can use g:termdebug_config['asm_winwidth'] and g:termdebug_config['var_winwidth'].  The default for these can be 60.

Reply:
" 60 is approx spaceBuffer * 3
it is fixed **minimal** value according to 'spaceBuffer' (of 'Var').